addons/sale_mrp/tests/test_sale_mrp_flow.py

```diff
@@ -1720,6 +1720,159 @@ def test_13_so_return_kit(self):
                                                            " company was successful, and the client is left without his 1 product.")
         self.assertEqual(qty_del_return_validated, 0.0, "The return has processed, client doesn't have any quantity anymore")
 
+    def test_14_anglo_saxon_variant_price_unit(self):
+        """
+        Test the price unit of a variant from which template has another variant with kit bom.
+        Products:
+            Template A
+                variant NOKIT
+                variant KIT:
+                    Component A
+        Business Flow:
+            create products and kit
+            create SO selling both variants
+            validate the delivery
+            create the invoice
+            post the invoice
+        """
+
+        # Create environment
+        self.env.company.currency_id = self.env.ref('base.USD')
+        self.uom_unit = self.UoM.create({
+            'name': 'Test-Unit',
+            'category_id': self.categ_unit.id,
+            'factor': 1,
+            'uom_type': 'bigger',
+            'rounding': 1.0})
+        self.company = self.env.ref('base.main_company')
+        self.company.anglo_saxon_accounting = True
```

Comment:
`uom_unit` is already defined in the `setUpClass`, and I think you can directly access the company through the env like you did for `currency_id`
```suggestion
        self.env.company.anglo_saxon_accounting = True
```
Makes no real difference except that shortens a little bit the test.
Otherwise, that's good, thank you :+1:

Reply:
noted, thanks. Committed and pushed